apmapf.py

```diff
@@ -230,21 +315,49 @@ def run(self):
 
     parser = argparse.ArgumentParser(prog='APMAPF',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('experiment', help='Experiment json path')
-    parser.add_argument('-t', '--timeout', help='Assume UNSAT for a given horizon after timeout (s)', default=0, type=int)
-    parser.add_argument('-v', '--verbosity', help='Verbosity level', default=2, type=int)
-    parser.add_argument('-m', help='Horizon multiplier', default=2.0, type=float)
-    parser.add_argument('-s', '--search', help='Path length optimization strategy.', default='linear', choices=['linear','binary'])
+
```

Comment:
Basically I just linted all this to an 80 character width limit per PEP08.  Feel free to ignore/undo if you disagree.

Reply:
PEP08 is the gospel truth!